simple_upload_server.go

```diff
@@ -20,6 +20,7 @@ func run(args []string) int {
 	maxUploadSize := flag.Int64("upload_limit", 5242880, "max size of uploaded file (byte)")
 	tokenFlag := flag.String("token", "", "specify the security token (it is automatically generated if empty)")
 	logLevelFlag := flag.String("loglevel", "info", "logging level")
+	corsEnable := flag.String("cors_enable", "false", "set to true to enable cors")
```

Comment:
use [`flag.Bool`](https://golang.org/pkg/flag/#Bool)

Reply:
I will do the changes as suggested and I'll try again. thanks